--- pg_hba.conf	18 Feb 2019 14:38:47 -0000	1.1
+++ pg_hba.conf	18 Feb 2019 17:14:47 -0000
@@ -83,9 +83,10 @@
 # "local" is for Unix domain socket connections only
 local   all             all                                     trust
 # IPv4 local connections:
-host    all             all             127.0.0.1/32            trust
+#host    all             all             127.0.0.1/32            trust
+host    all             galaxy           0.0.0.0/0            trust
 # IPv6 local connections:
-host    all             all             ::1/128                 trust
+#host    all             all             ::1/128                 trust
 # Allow replication connections from localhost, by a user with the
 # replication privilege.
 #local   replication     postgres                                trust
